compiler/jsgen.nim

```diff
@@ -1634,16 +1634,8 @@ proc genRepr(p: PProc, n: PNode, r: var TCompRes) =
   of tyEnum, tyOrdinal:
     gen(p, n.sons[1], r)
     useMagic(p, "cstrToNimstr")
-    var offset = ""
-    if t.kind == tyEnum:
```

Comment:
Well that code wasn't added without purpose. Check the git log to see the reason for this way of doing it.